modules/installation-creating-gcp-bootstrap.adoc

```diff
@@ -0,0 +1,106 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_gcp_user_infra/installing-gcp-user-infra.adoc
+
+[id="installation-creating-gcp-bootstrap_{context}"]
+= Creating the bootstrap machine in GCP
+
+You must create the bootstrap machine in Google Cloud Platform (GCP) to use during
+{product-title} cluster initialization. The easiest way to create this machine is
+to modify the provided Deployment Manager template.
+
+[NOTE]
+====
+If you do not use the provided Deployment Manager template to create your bootstrap
+machine, you must review the provided information and manually create
+the infrastructure. If your cluster does not initialize correctly, you might
+have to contact Red Hat support with your installation logs.
+====
+
+.Prerequisites
+
+* Configure a GCP account.
+* Generate the Ignition config files for your cluster.
+* Create and configure a VPC and assocated subnets in GCP.
+* Create and configure networking and load balancers in GCP.
+* Create control plane and compute roles.
+
+.Procedure
+
+. Copy the `04_bootstrap.py` template from the *Deployment Manager template for the bootstrap machine*
+section of this topic and save it as a YAML file on your computer. This template
+describes the bootstrap machine that your cluster requires.
+
+. Export the following variables required by the resource definition:
++
+----
+$ export CONTROL_SUBNET=`gcloud compute networks subnets describe ${INFRA_ID}-master-subnet --region=${REGION} --format json | jq -r .selfLink`
+$ export CLUSTER_IMAGE=`gcloud compute images describe ${INFRA_ID}-rhcos-image --format json | jq -r .selfLink`
+$ export ZONE_0=`gcloud compute regions describe ${REGION} --format=json | jq -r .zones[0] | cut -d "/" -f9`
+$ export ZONE_1=`gcloud compute regions describe ${REGION} --format=json | jq -r .zones[1] | cut -d "/" -f9`
+$ export ZONE_2=`gcloud compute regions describe ${REGION} --format=json | jq -r .zones[2] | cut -d "/" -f9`
+----
+
+. Create a bucket and upload the `bootstrap.ign` file:
++
+----
+$ gsutil mb gs://${INFRA_ID}-bootstrap-ignition
+$ gsutil cp bootstrap.ign gs://${INFRA_ID}-bootstrap-ignition/
+----
+
+. Create a signed URL for the bootstrap instance to use to access the Ignition
+config. Export the URL from the output as a variable:
++
+----
+$ export BOOTSTRAP_IGN=`gsutil signurl -d 1h service-account-key.json \
+    gs://${INFRA_ID}-bootstrap-ignition/bootstrap.ign | grep "^gs:" | awk '{print $5}'`
```

Comment:
nit: I dunno what our prompt policy is, but line continuation usually gets a `$PS2` prefix, and that's usually `> `, which would make this:

```console
$ export BOOTSTRAP_IGN=`gsutil signurl -d 1h service-account-key.json \
>   gs://${INFRA_ID}-bootstrap-ignition/bootstrap.ign | grep "^gs:" | awk '{print $5}'`
```

I also prefer `$(...)` command substitution to the backtick form, because [it's easier to nest][1] (although that doesn't matter in the cases you have here).

[1]: https://pubs.opengroup.org/onlinepubs/9699919799/utilities/V3_chap02.html#tag_18_06_03

Reply:
IMHO: adding extra chars such as `>` on a multi-line command makes it difficult to cut-and-paste... thus introducing unnecessary opportunity for human error.

Reply:
> IMHO: adding extra chars such as > on a multi-line command makes it difficult to cut-and-paste... 

So use a shell excerpt instead of a console excerpt and drop the `$ ` `PS1` too?

Reply:
The format here is the current openshift-docs convention for prompts, so would need to be a larger discussion w/ the team.